### ISSUE TITLE
PartiallyFailed backups because of the completed jobs

### ISSUE THREAD

**Issue by ugur99** [vmware-tanzu/velero#6250]
**What steps did you take and what happened:**
Velero backups are partially failing because of the pod status errors, ie for the some pods that are in completed status.

**What did you expect to happen:**
Completed status is expected when `jobs/cronjobs` are the case, and I think velero/restic should not return error for these cases. 

**The following information will help us better understand what's going on**:

```
level=error msg="Error backing up item" backup=velero/backup-velero-full-backup-daily-20230509220026 error="pod is not in the expected status, name=db-backup-job-28058160-64cfq, namespace=test, phase=Succeeded: pod is not running" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/util/kube/pod.go:28" error.function=github.com/vmware-tanzu/velero/pkg/util/kube.IsPodRunning.func1 logSource="pkg/backup/backup.go:435" name=db-backup-28058160-64cfq
```

**Anything else you would like to add:**

**Environment:**

- Velero version (use `velero version`): 
```
Client:
	Version: v1.11.0
Server:
	Version: v1.11.0
```
- Kubernetes version (use `kubectl version`):
```
v1.23.7
```

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

**Comment by reasonerjt** [comment]
@Lyndon-Li please dup this issue as it's probably fixed already per our discussion.

**Comment by Lyndon-Li** [comment]
See #6182, this is a known problem and will be fixed in v1.11.1

**Comment by ugur99** [comment]
Thank you @Lyndon-Li!
